packages/earth-admin/src/pages-client/layers/details.tsx

```diff
@@ -19,36 +15,23 @@
 
 import * as React from 'react';
 import { useEffect, useState } from 'react';
-import { Router } from '@reach/router';
 
 import { LayerContext } from 'utils/contexts';
 import { encodeQueryToURL, setPage } from 'utils';
 import { getAllLayers, getLayer } from 'services/layers';
 import { useRequest } from 'utils/hooks';
-import { LayerList, LayerDetails, LayerEdit, LinkWithOrg } from 'components';
+import { LayerList, LayerDetails, LayerEdit } from 'components/layers';
+import { LinkWithOrg } from 'components/link-with-org';
 import { useAuth0 } from 'auth/auth0';
 import { AuthzGuards } from 'auth/permissions';
 import { ContentLayout, SidebarLayout } from 'layouts';
 
-const LAYER_DETAIL_QUERY = { include: 'references', select: 'references.name,references.id' };
+const LAYER_DETAIL_QUERY = {include: 'references', select: 'references.name,references.id'};
 const INIT_CURSOR_LOCATION = '-1';
 
 const PAGE_TYPE = setPage('Layers');
 
-export default function LayersPage( props ) {
-  return (
-    <Router>
-      <Page path="/">
-        <HomePage path="/"/>
-        <DetailsPage path="/:page"/>
-        <EditPage path="/:page/edit" newLayer={false}/>
-        <EditPage path="/new" newLayer={true}/>
-      </Page>
-    </Router>
-  );
-}
-
-function Sidebar( props: any ) {
+function Sidebar(props: any) {
```

Comment:
this Sidebar component is not used anymore

Reply:
will be removed when i add the detail page